server/utils/map.js

```diff
@@ -111,13 +111,18 @@ export function mapChallengeToLang(
     lang = 'en';
   }
   const translation = translations[lang] || {};
+  const isTranslated = (Object.keys(translation).length > 0);
```

Comment:
I thought `const translation = translations[lang] || {};` was an object containing the requested language, or a blank object if there was no translation.

`isTranslated` is derived from whether `translation` is a blank object or not. 

Where does it fall down?